modules/sb-install-service-catalog.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-service-catalog.adoc
+
+[id='sb-enable-service-catalog-{context}']
+= Enabling service catalog
+
+Service catalog is included by default in {product-title}, but you must you must
+enable it using the procedure below.
+
+.Procedure
+
+. Create the service catalog API server.
```

Comment:
In recent builds, the ServiceCatalogAPIServer object is created during initial cluster creation, but the managed state is set to "Removed".  To enable Service Catalog, admins will need to  `oc edit ServiceCatalogAPIServer` and change the `managedState` from `Removed` to `Managed`.  This could be done either by using `oc edit` or using the file you have listed here and `oc apply -f svcat-api-server.yaml`  I imagine the latter is more in keeping with how we instruct admins to do things like this?

Reply:
I don't know if one way is preferred over the other, but I'll go with the file method for now and we can always update to use oc edit if we think that's the better method.